src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+    loadbalance
+}
+```
+
+Note that **100.2.177.123** is the ClusterIP address of the `submariner-lighthouse-coredns` Service we verified earlier.
+
+### 2. Export Services Across Clusters
+
+At this point, we have enabled secure IP communication between the connected clusters and formed the cluster set infrastructure. However,
+further configuration is required in order to signify that a Service should be visible and discoverable to other clusters in the cluster
+set. This can be done by creating a `ServiceExport` object in each cluster within the namespace that the underlying Service resides in.
+When a `ServiceExport` is created, this will cause the multi-cluster Service to become accessible as `<service>.<ns>.svc.clusterset.local`.
+Similarly, deleting the `ServiceExport` will stop exporting the Service.
+
+This guide uses a simple nginx server for demonstration purposes.
+
+{{% notice note %}}
+In the example below, we create the nginx resources within the nginx-test namespace. Note that the namespace must be created in both
+clusters for service discovery to work properly.
+{{% /notice %}}
+
+#### Test ClusterIP Services
+
+A Kubernetes Service is an abstraction which defines a logical set of Pods running in your cluster, that all provide the same functionality.
+When created, each Service is assigned a unique IP address. This address is tied to the lifespan of the Service.
+Pods can be configured to talk to the Service, and know that communication to the Service will be automatically load-balanced out to some
+Pod that is a member of the Service. ClusterIP is the default ServiceType in Kubernetes, and is only accessible from within the cluster.
+Submariner makes it possible for a ClusterIP Service to be accessible across cluster boundaries.
+
+##### 1. Create an `nginx` Deployment on **cluster3**
+
+```bash
+$ kubectl config use-context cluster3
+Switched to context "cluster3".
+```
+
+```bash
+$ kubectl create namespace nginx-test
+namespace/nginx-test created
+
+$ kubectl -n nginx-test create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
+deployment.apps/nginx created
+
+$ kubectl -n nginx-test expose deployment nginx --port=8080
+service/nginx exposed
+```
+
+This will create an `nginx` Service on the nginx-test namespace which targets TCP port 8080 on any Pod with the `app: nginx` label, and
+expose it on an abstracted Service port. When created, the Service is assigned a unique IP address (also called ClusterIP):
+
+```bash
+$ kubectl -n nginx-test get service nginx
+NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+nginx   ClusterIP   100.3.220.176  <none>        8080/TCP   2m41s
+```
+
+```bash
+$ kubectl -n nginx-test get pods -l app=nginx -o wide
+NAME                     READY   STATUS    RESTARTS   AGE   IP         NODE                NOMINATED NODE   READINESS GATES
+nginx-667744f849-t26s5   1/1     Running   0          3m    10.3.0.5   cluster3-worker2    <none>           <none>
```

Comment:
I assume these commands are to verify the service was created and running. We should preface these commands stating that.